stackoverflow-dark.css

```diff
@@ -163,11 +163,16 @@
     border-color: #555 !important;
     color: #ddd !important;
   }
+  
+  /* === herobox when not logged in === */
+#herobox.alternate {
+    background-color: rgba(0, 0, 0, 0.2) !important;
+}
 
   #header #topbar, .revision-comment, .date-span, .post-tag, .form-submit input, .preferred-login,
   .mdhelp, .wmd-help-button.active-help, .date_brick, .more-info, .review-stats-current-user,
-  .review-stats-count-current-user {
-    background: #222 !important;
+  .review-stats-count-current-user, .topbar.newheader, .topbar-menu-links, .network-items>a, .icon-help, .icon-help:hover, a:nth-child(2):hover  {
+    background-color: #222 !important;
```

Comment:
Actually, please don't add a `-color` to the background... it is meant to override the background image. You can move whatever definitions that were here up to line 91.